tokio/src/io/async_fd.rs

```diff
@@ -700,6 +700,7 @@ impl<T: AsRawFd> AsyncFd<T> {
     /// concurrently with other methods on this struct. This method only
     /// provides shared access to the inner IO resource when handling the
     /// [`AsyncFdReadyGuard`].
+    /// This method is cancellation safe.
```

Comment:
Please add this information like for other methods. For example, `TcpStream::readable` does it like this:

```suggestion
    /// # Cancel safety
    ///
    /// This method is cancel safe. Once a readiness event occurs, the method
    /// will continue to return immediately until the readiness event is
    /// consumed by an attempt to read that fails with `WouldBlock` or
    /// `Poll::Pending`.
```